.travis.yml

```diff
@@ -0,0 +1,120 @@
+language: cpp
+sudo: true
+
+branches:
+  only:
+  - master
+
+cache:
+- directories:
+  - $HOME/.cache
+
+addons:
+  apt:
+    packages:
+      - gdb
+
+script:
+  - export AWS_KVS_LOG_LEVEL=2
+  - make
+  - ulimit -c unlimited -S
+  - timeout --signal=SIGABRT 20m ./tst/producer_test
```

Comment:
Might be too tight. We will adjust this later.

Reply:
:+1: I just moved to two hours.